### DISCUSSION TITLE
Regarding the issues that arise after upgrading the OpenSea version

### DISCUSSION THREAD

**Discussion by Gino-lzy** [ProjectOpenSea/opensea-js#1354] [Q&A]
After upgrading opensea js from 2.0.2 to 7.0.7, import {CK-ADDRESS, CK-RINKEBY-ADDRESS} from "opensea js/lib/contracts";
Import {encodeTransferCall} from "opensea js/lib/utils/schema";
Import {WyvernProtocol} from "wyvern js/lib/wyvernProtocol"; These methods cannot be found. Is there an alternative method available
![Snipaste_2024-01-09_15-17-43](https://github.com/ProjectOpenSea/opensea-js/assets/71478954/338ad712-2c46-4da0-82e3-b99bf73f0a74)

**Comment by ryanio** [comment]
OpenSea no longer uses wyvern as of June 2022 so it is no longer supported. Rinkeby network is also deprecated, use Sepolia

encodeTransferCall, no longer, but there is a transfer() helper method if you need.
